modules/govuk_jenkins/manifests/jobs/populate_missing_data_etl.pp

```diff
@@ -0,0 +1,22 @@
+# == Class: govuk_jenkins::jobs::populate_missing_data_etl
+#
+# Create a jenkins job to periodically run rake for the following tasks:
+# - rake etl:rerun_master
+#
+# === Parameters:
+#
+# [*rerun_rake_etl_master_process_cron_schedule *]
+#   The cron timings for the etl:master process
+#   Default: undef
+#
+class govuk_jenkins::jobs::populate_missing_data_etl (
+  $rerun_rake_etl_master_process_cron_schedule = undef,
```

Comment:
```suggestion
  $rerun_rake_etl_master_process_cron_schedule,
```

Since it's always set we could make it a required argument.

Reply:
I get some test fails when trying this: 
```
`Puppet::Error:
       Must pass re_run_rake_etl_master_process_cron_schedule to Class
```

https://ci.integration.publishing.service.gov.uk/job/govuk-puppet/job/add-rerun-etl-job/11/console

Would i need to change anything else to make it work?

Reply:
Hmm, I'm not sure exactly how the testing works in this repo, but I think you might need to add a value to https://github.com/alphagov/govuk-puppet/blob/master/spec/fixtures/hieradata/common.yaml ?